src/app/dataset-create/dataset-create.component.ts

```diff
@@ -45,7 +47,8 @@ export class DatasetCreateComponent extends BaseComponent {
         visibility: [DatasetVisibility.Private],
     });
     public readonly DROPDOWN_LIST: SelectStorageItemType[] = STORAGE_LIST;
-    public selectedStorage: number;
+
+    public selectedStorage: number = 1;
```

Comment:
What is 1? At least a comment. If I change anything in the collection, it wont' survive.

Reply:
It's default id item